### ISSUE TITLE
chinese caracter with two prononciation

### ISSUE THREAD

**Issue by flyingsoul** [batterseapower/pinyin-toolkit#146]
Hi, I try to find a solution on internet but didn't found so sorry if I just miss the solution.
When I put for exemple  号 in the expression field the meaning is roar with háo as pinyin how to get other meaning for the different tone like hào wich is the date in oral language but the same chinese caracter or an other exempler 累 lěi means accumulate but 累 lèi means tired.
thanks and congratulation for your job it's very nice plugin :)

**Comment by batterseapower** [comment]
Yes, this is an important feature. I had exactly the same problem with lei just the other day in fact :-)

What I do to work around it is keep the CE DICT open in a text editor and copy and paste in the correct definition. However, it would be much nicer if you could choose which one you meant somehow. I'm not quite sure what that should look like, though...

**Comment by Nick3C** [comment]
Yes, I still don't have a really satisfactory answer for how to deal with this...

How about this as a real and sensible suggestion: where there are multiple matches, we could add a button with a double arrow (>>) on it to the right-hand side of the "expression" field (or elsewhere). Clicking on the button would bring up a menu and the menu would list matching cedict entries. Each entry would show meaning, pinyin, simp/trad hanzi [depending on preference]. Clicking on any entry would cause the data in the fields to be updated to match that entry.

That would seem to add on to the current behaviour without changing the way the toolkit fundamentally works. It would also default to simply show one of the entries (and thus be right a portion of the time).

At a later point we could also work on a way to preference some entries over others to default to the most frequently used (perhaps a google search for "[character] + [english]" favouring the entry with the larger number of results

**Comment by tbbhoiorg** [comment]
I was just wondering, I've been having the same problem, and I was looking for a solution. I notice this discussion all takes place two years ago, so was a solution ever found? For what it's worth, Nick3C's solution seems perfect.
Thanks
G
